pandas/tests/test_stats.py

```diff
@@ -108,6 +108,31 @@ def test_rank_dense_method(self):
                 expected = Series(exp).astype(result.dtype)
                 assert_series_equal(result, expected)
 
+        # GH15630, pct should be on 100% basis even when method='dense'
```

Comment:
make this a parametrized tests (needs to be a separate function, not in this class)